### PR TITLE
Add Note component

### LINKED ISSUE #127: Note component
Mockup:

image

A `Note` component has:

- Avatar
- Full name
- Date (prop should be a `Date` object, and we will render as a relative time, eg. _"1 h ago"_ or _"10 m ago"_
- Body (render as markdown)

## Behaviour

- if the Date is less than 1 minute ago, it should always be rendered as _"just now"_
- if the Date is more than 24 hours ago, it should be rendered as "dd mmm  yy"
- the Date should be periodically re-rendered, so that as time passes the correct relative time is shown.

### PR DESCRIPTION
**Pull request by bernardodiasc**
Relates to the part of #127 

Please run `npm install` for new dependencies before run on this branch.

## Preview

screen shot 2018-05-24 at 2 39 10 pm

---

Thanks @skvale for the great work. The code was really good and polished, I just included the existing Avatar component in place.

### CODE REVIEW

**Comment by nicksp** [comment]
I agree that the original code was of a really good quality.

**Comment by nicksp** [comment]
@skvale You're right. We'll fix that. We're working on that actually. Thanks for the contribution!

**Comment by nicksp** [comment]
@bernardodiasc As discussed, please investigate the possibility to re-use existing `MarkdownEditor` component instead of introducing a new dependency.

**Comment by bernardodiasc** [comment]
I believe no action is required, after taking a look at this possibility I concluded that the `medium-editor` is not purposed to be used as a markdown renderer. On other hand `markdown-to-jsx` is one of the best (if not the best) lib to deal with this, introducing this dependency can be valuable if we need more powerful text rendering options in the future.

File: src/components/ui/Text.js [hunk 1]
```diff
@@ -43,7 +44,19 @@ const Content = elem.div(cmz(
...
+const PureContent = elem.span(cmz(
+  typo.baseText,
+  `
+    a {
+      color: ${theme.baseRed}
+      text-decoration: none
+    }
+
+    a:hover {
+      text-decoration: underline
```
Comment (nicksp) [inline src/components/ui/Text.js]: By introducing this global change you broke UI of some of the admin pages where PureComponent is used. On hover, links became underlined with red where it shouldn't. Be careful next time introducing global changes for existing components! Reverted that change in `master`.

**Pull request merged**